runtime/compiler/optimizer/FearPointAnalysis.cpp

```diff
@@ -35,16 +35,26 @@
 #include "infra/Checklist.hpp"
 
 
-bool TR_FearPointAnalysis::virtualGuardsKillFear(TR::Compilation *comp, TR::Node *virtualGuardNode)
+bool TR_FearPointAnalysis::virtualGuardKillFear(TR::Compilation *comp, TR::Node *virtualGuardNode)
```

Comment:
Sorry to ask you to change this again, but it reads strangely now because `Kill` is still conjugated for plural guards. Could I bother you to make it `Kills` (with an `s`)? i.e. it's true when the guard *kills* fear

Reply:
Changed in https://github.com/eclipse-openj9/openj9/pull/16354/commits/d2a57e7eaec47151b5e7794df65185ae1cd5fead